LibreHardwareMonitorLib/Hardware/Cpu/Amd17Cpu.cs

```diff
@@ -354,6 +365,19 @@ public void UpdateSensors()
                     _busClock.Value = (float)(_cpu.TimeStampCounterFrequency / timeStampCounterMultiplier);
                     _cpu.ActivateSensor(_busClock);
                 }
+
+                if (_cpu._smu.IsPmTableLayoutDefined())
+                {
+                    var smuData = _cpu._smu.GetPmTable();
+                    foreach(var sensor in _smuSensors)
+                    {
+                        if(smuData.Length >= sensor.Key)
```

Comment:
Formatting.

Reply:
Can you please give a sample?

Reply:
If you'd run VS' formatting it should do this automatically based on the editor config. In here, there are no spaces after foreach/if (there are more of these).

Reply:
Thanks, fixed.